src/api/bannerAdmin.ts

```diff
@@ -1,16 +1,29 @@
+import Bugsnag from "@bugsnag/js";
 import firebase from "firebase/app";
+import { Banner } from "types/banner";
 
-export const makeUpdateBanner = (
-  venueId: string,
-  onError: (errorMsg: string) => void
-) => async (message?: string): Promise<void> => {
+export interface MakeUpdateBannerProps {
+  venueId: string;
+  banner?: Banner;
+}
+
+export const makeUpdateBanner = async ({
+  venueId,
+  banner,
+}: MakeUpdateBannerProps): Promise<void> => {
   const params = {
     venueId,
-    bannerMessage: message ?? "",
+    banner: banner ?? firebase.firestore.FieldValue.delete(),
   };
 
   await firebase
     .functions()
     .httpsCallable("venue-adminUpdateBannerMessage")(params)
-    .catch((e) => onError(e.toString()));
+    .catch((err) =>
+      Bugsnag.notify(err, (event) => {
+        event.addMetadata("api/bannerAdmin::makeUpdateBanner", {
+          venueId,
```

Comment:
I think it would be nice if you pass `banner` here as well.

Reply:
fixed 42fc0f3